pkg/ingester/ingester.go

```diff
@@ -95,6 +95,7 @@ const (
 	sampleOutOfBounds    = "sample-out-of-bounds"
 
 	replicationFactorStatsName             = "ingester_replication_factor"
+	ringKVStoreStatsName                   = "ingester_ring_kvstore"
```

Comment:
Confused about this not being `ingester_ring_store` to mimic the flag, but it's not important so it's ok to me.

Reply:
Right. In YAML it's called `kvstore`, but it's called `store` in CLI flag. I don't have a preference neither. I can rename it to `*_store`.

Reply:
Done in https://github.com/grafana/mimir/pull/2981/commits/eace8333001f17445fceac952558f3d61b9196bc.